geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegion.java

```diff
@@ -611,7 +611,8 @@ PartitionedRegionRedundancyTracker getRedundancyTracker() {
     return redundancyTracker;
   }
 
-  public void computeWithPrimaryLocked(Object key, Runnable r) throws PrimaryBucketLockException {
+  public boolean computeWithPrimaryLocked(Object key, Runnable r, boolean tryLock)
```

Comment:
My understanding is that we no longer use this new "tryLock" parameter. So should we remove it from this method so that this PR does not change this code?

Reply:
You are correct, Darrel, we no longer need it.  Jens and I talked about it a bit this. morning and thought it might be beneficial to have the option in the method, but since it's not needed for this PR it probably makes sense to remove it.